client/me/purchases/purchases-list/header/index.jsx

```diff
@@ -35,6 +35,12 @@ const PurchasesHeader = ( { section } ) => {
 					{ i18n.translate( 'Billing History' ) }
 				</NavItem>
 
+				{ config.isEnabled( 'async-payments' ) && (
+					<NavItem path={ purchasesRoot + '/pending' } selected={ section === 'pending' }>
+						{ i18n.translate( 'Pending Payments' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Pending payment' )` *ES Score: 12*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/pending-payments)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).